app/main.py

```diff
@@ -16,4 +16,6 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "mutable": [my_favourite_films, marks, collection_of_coins],
+    "immutable": [lucky_number, pi, one_is_a_prime_number, name, profile_info]}
```

Comment:
Bad code style: Your opening brace is separate but the closing one isn't. Remake it one style